front/app/containers/Admin/formBuilder/components/FormFields/index.tsx

```diff
@@ -93,7 +93,7 @@ const FormFields = ({
                         </Text>
                       </Box>
                       {!isNilOrError(field.input_type) && (
-                        <StyledBadge color={colors.adminSecondaryTextColor}>
+                        <StyledBadge className="inverse" color={colors.grey}>
```

Comment:
Is this new class name being used for styling somewhere? I might have missed it, but I don't think it's in this file?

Reply:
It's inverses the colors so that the `color` attribute controls the background color instead of the text color.

Reply:
Ah I see! Ok thanks :)